### ISSUE TITLE
ColloquialsinceNow crash

### ISSUE THREAD

**Issue by iantheparker** [malcommac/SwiftDate#283]
This code here

```
do {
      print(card.publishedDate()) // prints "2016-09-29 22:59:57 +0000"
>>>>>>let (colloquial, _) = try card.publishedDate().colloquialSinceNow()
      ageLabel.text = colloquial
} catch {
      ageLabel.text = card.published_at
}
```
is crashing here
![image](https://cloud.githubusercontent.com/assets/1368966/19120273/4dfce0f8-8af0-11e6-9dad-0c8ed9580d5d.png)

Any ideas?

**Comment by meandkareem** [comment]
I have same issue on same line.
fatal error: unexpectedly found nil while unwrapping an Optional value

**Comment by zackz-woolloo** [comment]
same issue here, the cmp.weekOfYear is nil

**Comment by malcommac** [comment]
Closed and scheduled for 4.0.4

**Comment by raven** [comment]
So this is still occurring in 4.0.4
